ios/EnrichedTextInputView.mm

```diff
@@ -529,6 +529,8 @@ - (void)updateProps:(Props::Shared const &)props
     stylePropChanged = YES;
   }
 
+  NSRange prevSelectedRange = textView.selectedRange;
```

Comment:
Why don't we put it inside of the `if (stylePropChanged` block? I'm afraid such code out of the context could break something in the future. Let's keep it in its context.